### PR TITLE
made home screen look nice, added button component, small changes els…

### PR DESCRIPTION
**Pull request by ardymarky**
…ewhere

### CODE REVIEW

File: app/buy/index.js [hunk 1]
```diff
@@ -42,6 +42,7 @@ const Buy = () => {
         <Page header="Select Location" style={styles.page}>
             <ScrollView contentContainerStyle={styles.scrollContainer}>
                 {options.map((option) => (
+                    // <View style={styles.location} key={option.label}>
```
Comment (TheGoldenPatrik1) [inline app/buy/index.js]: Can we remove these comments or is there a reason they're still here?

File: app/index.js [hunk 2]
```diff
@@ -1,56 +1,69 @@
...
+        <Page header={`Hello ${name}`} style={styles.page}>
             <View style={styles.buttonContainer}>
-                <Text>{name}Would you like to...</Text>
-                <Button mode="contained" onPress={onBuyPress}>
-                    Buy
-                </Button>
-                <Button mode="contained" onPress={onSellPress}>
-                    Sell
-                </Button>
+                <Text style={styles.question}>{name}Would you like to...</Text>
```
Comment (TheGoldenPatrik1) [inline app/index.js]: Shouldn't we change this to `{name} would you like to...`

File: app/index.js [hunk 3]
```diff
@@ -1,56 +1,69 @@
...
+    divider: {
+        width: '40%', // Divider spans full width of the screen
+        height: 1, // Thin divider
+        backgroundColor: '#B3B3B3', // Light gray color for the divider
+        marginVertical: 15 // Spacing between links and dividers
+    },
+    question: {
+        fontSize: 24,
+        marginBottom: '3%',
+        color: '9E1B32',
```
Comment (TheGoldenPatrik1) [inline app/index.js]: Does this color work without a `#` in front?

File: components/Button.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        justifyContent: 'center'
+    },
+    buttonText: {
+        color: '#9E1B32',
+        fontSize: 30,
+        fontWeight: '500'
+    }
+})
+
+Button.propTypes = {
```
Comment (TheGoldenPatrik1) [inline components/Button.js]: Make all of these props required with `string.isRequired`

File: app/index.js (hunk 3 above)
Comment (lmillet3804) [reply app/index.js]: do we have some kind of global color vars defined?

Comment (TheGoldenPatrik1) [reply app/index.js]: We do, but they rely on the react native paper `useTheme` hook, which I think you can only call inside a render method (maybe I'm wrong though)

Comment (lmillet3804) [reply app/index.js]: ah, okay

File: app/buy/index.js (hunk 1 above)
Comment (ardymarky) [reply app/buy/index.js]: smh

File: app/index.js (hunk 2 above)
Comment (ardymarky) [reply app/index.js]: smh

File: app/index.js (hunk 3 above)
Comment (ardymarky) [reply app/index.js]: smh

File: components/Button.js (hunk 4 above)
Comment (ardymarky) [reply components/Button.js]: smh

**Review by TheGoldenPatrik1 — APPROVED** [review]

**Review by lmillet3804 — APPROVED** [review]

**Pull request merged**